source/numbers.rst

```diff
@@ -42,6 +42,7 @@ You just type them as literals::
 Questions
 ---------
```

Comment:
Thats cool! Okay, I will put in a question so people can see the difference between `5/2` and `5/2.0`. Thanks @akash1810 and everyone :)